tests/unit/lms/security_test.py

```diff
@@ -407,6 +407,7 @@ def test_get_policy_google(self, pyramid_request, path, LMSGoogleSecurityPolicy)
             ("/content_item_selection", get_lti_user_from_launch_params),
             ("/canvas_oauth_callback", get_lti_user_from_oauth_callback),
             ("/api/blackboard/oauth/callback", get_lti_user_from_oauth_callback),
+            ("/api/canvas_studio/oauth/callback", get_lti_user_from_oauth_callback),
             ("/api/d2l/oauth/callback", get_lti_user_from_oauth_callback),
```

Comment:
It would be nice not to have to declare each of the callback routes individually here. Perhaps we can just match on a `/oauth/callback` suffix in the path?

Reply:
The previous version of the security module just tried every possible policy sequentially so that always "worked" but it was a debugging nightmare.

Maybe we are now on the other side of the spectruct having to declare the path explicitly. 

We have a .starswith already on security.py, we could do a `endswith` if needed but we don't often change these.

I like the "greppeablity" of the current approach for now.

Reply:
> I like the "greppeablity" of the current approach for now.

OK. I'll leave this as-is for the moment.